compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -6020,8 +6150,11 @@ TR::Node *iaddSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
       {
       return node;
       }
+   auto binOpSimplifier = getIntBinaryOpSimplifier(s);
```

Comment:
Could an empty line be added before the change for better readability and being consistent with the original code format which has an empty line before `BINARY_IDENTITY_OP`?

 The same comment applies to other places where the new APIs are called.